plugins/API/lang/en.json

```diff
@@ -1,5 +1,6 @@
 {
     "API": {
+        "ChangeTokenHint": "If you want to change this token, please go to your personal settings page.",
```

Comment:
* would be useful to add a link to the settings page: `href="index.php?module=UsersManager&action=userSettings"`